src/basics.md

```diff
@@ -111,9 +111,11 @@ fn decode(mut bytes: &[u8]) -> Result<Payload> {
 # quick_main!(run);
 ```
 
-[ex-rand-float]: #ex-rand-float
-<a name="ex-rand-float"></a>
-## Generate random floating point numbers
+[ex-rand-simple]: #ex-rand-simple
+<a name="ex-rand-simple"></a>
+## Generate random numbers for simple types
+
+The `rand` crate provides a convenient source of psuedo-random numbers.
```

Comment:
I originally wanted to suggest removing the  whole
>"The `rand` crate provides a convenient source of psuedo-random numbers"

as all of that information is either already available (as you have noted) or obvious